driver-scala/src/main/scala/org/mongodb/scala/MongoCredential.scala

```diff
@@ -192,7 +192,7 @@ object MongoCredential {
    * @param source   the source where the user is defined.  This can be either `\$external` or the name of a database.
    * @param password the non-null user password
    * @return the credential
-   * @see [[http://docs.mongodb.org/manual/core/authentication/#ldap-proxy-authority-authentication PLAIN]]
+   * @see [[https://www.mongodb.com/docs/manual/core/authentication/#ldap-proxy-authority-authentication PLAIN]]
```

Comment:
Looks like this refers to an old target.
New link suggestion: https://www.mongodb.com/docs/manual/core/security-ldap/